features/step_definitions/content.rb

```diff
@@ -16,6 +16,13 @@
   expect(page).to have_link(text)
 end
 
+When(/^he (un)?checks the checkbox '(.*)'$/) do |un, checkbox_name|
```

Comment:
How about highlighting `(unchecked|checked)` and binding it to parameter `action`?